estate/__manifest__.py

```diff
@@ -2,7 +2,7 @@
     'name': "estate",
     'version': '0.1',
     'depends': ['base'],
-    'data':[
```

Comment:
As a git exercise, try to make the small typo changes inside the commits where the line was initially introduced.
You can look into the `git rebase -i` command to see how to rewrite your branch's history.
You will need to use `git push --force-with-lease` to send a rewritten version of your branch. (Try to avoid using `git push --force` to avoid accidents in situations where you work on a common branch with other people)